api/src/main/java/io/grpc/MethodDescriptor.java

```diff
@@ -51,7 +51,8 @@
 
   // Must be set to InternalKnownTransport.values().length
   // Not referenced to break the dependency.
-  private final AtomicReferenceArray<Object> rawMethodNames = new AtomicReferenceArray<>(1);
+  private final AtomicReferenceArray<Object> rawMethodNames =
+      new AtomicReferenceArray<>(InternalKnownTransport.values().length);
```

Comment:
This will break Bazel; use a literal `2`.

The comment alludes to why this was done, but even if you thought that no longer applied the comment should have been removed.

Reply:
`InternalKnownTransport` is part of grpc-api so Bazel seems not impacted. It breaks blaze though.

Reply:
Just noticed there were already comments on this field
```java
// Must be set to InternalKnownTransport.values().length
// Not referenced to break the dependency.
private final AtomicReferenceArray<Object> rawMethodNames =
```